_layouts/landing_page.html

```diff
@@ -26,34 +26,104 @@ <h1>Hyperledger Grid</h1>
+                    <img src="{% link assets/splinter_logos_logotype_gradient.svg %}"
+                    height="24" alt="Splinter">
+                </h1>
+            </div>
+            <div id="landing-page-lower-section" class="landing-page-lower-section">
+                <p>
+                    Grid lets you combine a distributed ledger framework with
+                    Hyperledger components and Grid smart contracts into a
+                    single, effective business solution.
+                    Grid's flexible, modular design supports a choice of backend
+                    distributed ledgers:
+                    <a href="https://www.splinter.dev/">Splinter</a> and
+                    <a href="https://sawtooth.hyperledger.org/">Hyperledger
+                    Sawtooth</a>.
+                </p>
+                <p>
+                    Hyperledger's
+                    <a href="https://sawtooth.hyperledger.org/docs/sabre/releases/latest/">
+                    Sawtooth Sabre</a> is the enabling technology behind
+                    Grid's support for multiple distributed ledgers.
+
+                    Sabre provides a smart contract engine for customized smart
+                    contracts and the included smart contracts in Grid:
+                    <ul>
+                        <li>
+                            <b>Location</b> is a framework for sharing location
+                            master data between trade partners.
+                        </li>
+                        <li>
+                            <b>Product</b> provides a way to share
+                            GS1-compatible product data (items that are
+                            transacted, traded, or referenced in a supply
+                            chain).
+                        </li>
+                        <li>
+                            <b>Pike</b> is a Grid smart contract that handles
+                            organization and identity permissions with Sabre.
+                        </li>
+                        <li>
+                            <b>Schema</b> is a Grid smart contract that provides
+                            a reusable, standard approach to defining, storing,
+                            and consuming product properties.
+                        </li>
+                    </ul>
+                </p>
+                <p>
+                    Learn more > &nbsp;
+                    <img src="{% link assets/splinter_logo.svg %}"
+                    width="48" height="48" alt="Splinter logo">
+                    <a href="/docs/{{ site.data.general.latest_version
+                    }}/grid_on_splinter.html">Running Grid on Splinter</a>
+                    &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;
+                    <img src="https://sawtooth.hyperledger.org/img/avatar.png"
+                    width="36" height="36" alt="Hyperledger Sawtooth logo">
+                    Running Grid on Sawtooth
```

Comment:
This can also be a link now that the page exists for running on sawtooth.

Reply:
updated